macros/cross_db_utils/length.sql

```diff
@@ -0,0 +1,39 @@
+{% macro length(expression) -%}
+    {{ adapter_macro('dbt_utils.length', expression) }}
+{% endmacro %}
+
+
+{% macro default__length(expression) %}
```

Comment:
this would be shorter if you made the default `length({{expression}})` and then just made a redshift version that differed from that :) functionally equivalent, less code!